src/main/java/com/matera/cadastrocentral/telephone/TelephoneService.java

```diff
@@ -83,7 +87,7 @@ public Telephone patchTelephonePropertyByTelephoneId(UUID telephoneId, Telephone
             Telephone auxTelephone = optionalTelephone.get();
```

Comment:
I think that in the line above, and in the block of code below, maybe you should throw some exceptions, for example, what happens if the optionalTelephone is null?